gc/base/MemorySubSpaceUniSpace.cpp

```diff
@@ -443,17 +447,19 @@ MM_MemorySubSpaceUniSpace::calculateExpandSize(MM_EnvironmentBase *env, uintptr_
 {
 	uintptr_t currentFree, minimumFree, desiredFree;
 	uintptr_t expandSize = 0;
+	uintptr_t heapFreeMinimumHeuristicMultiplier;
```

Comment:
Bring down this definition and (now that you touching the code) any other from this group to the site they are first time initialized.